packages/oidc-client/__tests__/main.test.ts

```diff
@@ -0,0 +1,32 @@
+var httpclient = require('@actions/http-client')
+var configvar = require('./../src/internal/config-variables')
+var main = require('./../src/main')
+
+describe('oidc-client-tests', () => {
+  it('Get Http Client', async () => {
+    const http = new httpclient.HttpClient('actions/oidc-client')
+    expect(http).toBeDefined()
+  })
+
+  it('HTTP get request to get token endpoint', async () => {
+    const http = new httpclient.HttpClient('actions/oidc-client')
+    const res = await http.get(
+      'https://ghactionsoidc.azurewebsites.net/.well-known/openid-configuration'
```

Comment:
@souravchanduka Do we intend to take dependancy on the POC instance and not the GH actions instance for this (at least temporarily)?
cc: @navin22

Reply:
no, we should remove this. L0 should be fine for this. And if there are e2e tests we should follow the existing infra to call the apis.